infos/unileben.md

```diff
@@ -0,0 +1,227 @@
+Versicherungspflicht nötig, die es nach einer Aufklärung bei einer
+beliebigen gesetzlichen Krankenkasse gibt. Achtung: Ist diese Befreiung
+erst einmal ausgestellt, gibt es kein zurück mehr\! Erst eine mindestens
+einjährige (Vollzeit-)Beschäftigung berechtigt wieder zum Eintritt in
+eine gesetzliche Krankenkasse.
+
+Die Studententarife (fast) aller Krankenkassen gehen davon aus, dass man
+hauptsächlich studiert und nebenher höchstens ein bisschen arbeitet. In
+konkreten Zahlen heisst das:max. 20 Stunden pro Woche, [in den
+Semesterferien](https://www.versicherungswissen.org/studenten/20-stunden-regel-studenten.html)
+gilt keine Stundenbegrenzung. Für die Pflegeversicherung gilt die
+bekannte [345/400
+EUR-Grenze](https://www.tarifini.de/pflegeversicherung-fuer-studenten/).
+Je nach Krankenkasse gibt es auch ein Maximaleinkommen, bei dessen
+Überschreitung man aus dem Studententarif fliegt und in einen -
+wesentlich teureren - normalen Tarif kommt, der dann meist für ein
+volles Jahr gilt.
+
+## Literatur
+
+Fachliteratur ist auch in der Informatik oft nicht billig, allerdings
+ist man - im Gegensatz zu anderen Studienfächern - nicht auf Bücher als
+Hauptinformationsquelle angewiesen. Zudem steht die
+[Informatik-Bibliothek](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/fachbereich/bibliothek/) zur
+Verfügung. Desweiteren empfiehlt sich auch bei alternativen
+Buchportalen vorbeizuschauen.
+
+## Wohnung, Miete, Nebenkosten
+
+Die Region Tübingen/Reutlingen ist, was die Mietpreise angeht, eine der
+teuersten Regionen Deutschlands. Zudem ist es nicht immer ganz einfach,
+überhaupt eine Wohnung oder ein Zimmer zu finden. Für den privaten
+Wohnungsmarkt sind Mittwochs- und Samstagsausgabe des "[Schwäbischen
+Tagblatts](https://www.tagblatt.de)" zu empfehlen, natürlich die
+Schwarzen Bretter in den Mensen, und auch die "Studentische
+Zimmervermittlung" in der Mensa Wilhelmstraße ist sehr hilfreich bei der
+Zimmersuche. Neben privaten Wohnungen gibt es auch die Möglichkeit in
+einem Studentenwohnheim der beiden Studentenwerke zu wohnen.
+Antragsformulare für einen Wohnheimplatz gibt es entweder beim
+Studentenwerk AdöR in der Wohnheimverwaltung, Fichtenweg 5, 72076
+Tübingen, oder beim Studentenwerk e.V., Rümelinstraße 8, 72070
+Tübingen. Übrigens: Das erste, was man machen sollte, ist eine
+offlineversion eines Stadtplans zu kaufen. Old style paper version.
+Solltet ihr mit dem Auto nach Tübingen kommen, plant für jede Strecke am
+Anfang eine halbe Stunde mehr ein - nicht auf Grund des hohen
+Verkehrsaufkommens sondern der hohen Dichte an Einbahnstrassen. Es
+empfielt sich häufig das Prinzip: "Erst dran vorbei und dann von hinten
+durch die Mitte" ;-)
+
+## Essen in Tübingen
+
+### Frühstück
+
+``` 
+  * Gut und preiswert Frühstücken kann man im Stern (Lange Gasse). Von ca. 9 - 13 Uhr bekommt man dort einen Zettel auf dem man sich sein Frühstück zusammenkreuzen kann.
+```
```

Comment:
The code blocks appear pretty often in this file, I assume this is due to an automatic conversion.